op-node/rollup/derive/error.go

```diff
@@ -96,3 +96,6 @@ var ErrCritical = NewCriticalError(nil)
 // NotEnoughData implies that the function currently does not have enough data to progress
 // but if it is retried enough times, it will eventually return a real value or io.EOF
 var NotEnoughData = errors.New("not enough data")
+
+// EngineSyncing implies that the execution engine is currently in progress of P2P sync.
+var EngineSyncing = errors.New("engine is syncing")
```

Comment:
```suggestion
var EngineP2PSyncing = errors.New("engine is p2p syncing")
```
I suggest this change to distinguish between p2p sync and normal sync.